src/wing/CCPACSWing.h

```diff
@@ -64,125 +78,280 @@ friend class CTiglWingBuilder;
+     */
     TIGL_EXPORT CCPACSWingSegment& GetSegment(const int index);
+
+    /**
+     * @brief Returns the segment for a given index
+     * @param index Segment index
+     * @return CCPACSWingSegment&
+     */
     TIGL_EXPORT const CCPACSWingSegment& GetSegment(const int index) const;
+
+    /**
+     * @brief Returns the segment for a given uid
+     * @param uid Segment UID
+     * @return CCPACSWingSegment&
+     */
     TIGL_EXPORT CCPACSWingSegment& GetSegment(std::string uid);
+
+    /**
+     * @brief Returns the segment for a given uid
+     * @param uid Segment UID
+     * @return CCPACSWingSegment&
+     */
     TIGL_EXPORT const CCPACSWingSegment& GetSegment(std::string uid) const;
 
-    // Getter of the number of component segments of the wing
+    /**
+     * @brief Returns the number of component segments of the wing
+     * @return int
+     */
     TIGL_EXPORT int GetComponentSegmentCount() const;
 
-    // Returns the segment for a given index or uid
+    /**
+     * @brief Returns the segment for a given index or uid
+     * @param index Segment index
+     * @param uid Segment UID
+     * @return CCPACSWingComponentSegment&
+     */
     TIGL_EXPORT CCPACSWingComponentSegment& GetComponentSegment(const int index);
     TIGL_EXPORT const CCPACSWingComponentSegment& GetComponentSegment(const int index) const;
     TIGL_EXPORT CCPACSWingComponentSegment& GetComponentSegment(const std::string& uid);
 
-    // Get the positioning transformation for a given section uid
+    /**
+     * @brief Returns the positioning transformation for a given section uid
+     * @param sectionUID
+     * @return CTiglTransformation
+     */
     TIGL_EXPORT CTiglTransformation GetPositioningTransformation(std::string sectionUID);
 
-    // Gets the upper point in absolute (world) coordinates for a given segment, eta, xsi
+    /**
+     * @brief Returns the upper point in absolute (world) coordinates for a given segment,
+     * eta, xsi (calculated output may be influenced by setting different value for Enum getPointBehavior)
+     * @param segmentIndex
+     * @param eta
+     * @param xsi
+     * @return gp_nP
+     */
     TIGL_EXPORT gp_Pnt GetUpperPoint(int segmentIndex, double eta, double xsi);
 
-    // Gets the upper point in absolute (world) coordinates for a given segment, eta, xsi
+    /**
+     * @brief Returns the lower point in absolute (world) coordinates for a given segment, eta, xsi
+     * (calculated output may be influenced by setting different value for Enum getPointBehavior)
+     * @param segmentIndex
+     * @param eta
+     * @param xsi
+     * @returns gp_nP
+     */
     TIGL_EXPORT gp_Pnt GetLowerPoint(int segmentIndex, double eta, double xsi);
 
-    // Gets a point on the chord surface in absolute (world) coordinates for a given segment, eta, xsi
+    /**
+     * @brief Returns a point on the chord surface in absolute (world) coordinates for a given segment, eta, xsi, if fourth parameter is ommited
```

Comment:
This is really picky, but the grammer of that sentence can be misunderstood. What is being calculated, if the fourth parameter is not ommited?

Reply:
yes, I'll check that